### ISSUE TITLE
undefined method `translation_repositories' for GettextSetup:Module

### ISSUE THREAD

**Issue by ktreese** [puppetlabs/tasks-hands-on-lab#23]
I came across this issue:

```
➜  tasks bolt task run package action=status package=bash  --nodes 127.0.0.1:32768,127.0.0.1:32769,127.0.0.1:32770  --modulepath ./modules -u root -p root
/opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/module.rb:445:in `i18n_initialized?': undefined method `translation_repositories' for GettextSetup:Module (NoMethodError)
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/module.rb:427:in `initialize_i18n'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/module.rb:101:in `initialize'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/node/environment.rb:333:in `new'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/node/environment.rb:333:in `block in modules'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/node/environment.rb:331:in `collect'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/node/environment.rb:331:in `modules'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/node/environment.rb:288:in `module'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/module.rb:35:in `find'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/info_service/task_information_service.rb:18:in `task_data'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/lib/bolt/cli.rb:451:in `block in load_task_data'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet/context.rb:65:in `override'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/vendored/puppet/lib/puppet.rb:264:in `override'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/lib/bolt/cli.rb:450:in `load_task_data'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/lib/bolt/cli.rb:369:in `block in execute'
	from /opt/puppetlabs/puppet/lib/ruby/2.1.0/benchmark.rb:294:in `realtime'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/lib/bolt/cli.rb:357:in `execute'
	from /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/bolt-0.7.0/exe/bolt:9:in `<top (required)>'
	from /opt/puppetlabs/puppet/bin/bolt:23:in `load'
	from /opt/puppetlabs/puppet/bin/bolt:23:in `<main>'
```

I implemented the following workaround to resolve it:
https://tickets.puppetlabs.com/browse/SERVER-1912

**Comment by garethr** [comment]
@ktreese thanks for reporting. Could I just check, are you saying you installed the `gettext-setup` gem to resolve the issue? 

If you could open an issue against the `bolt` repository at https://github.com/puppetlabs/bolt/issues that would be much appreciated. Unfortunately Github doesn't allow for moving issues, and this repository mainly deals with the content for the hands-on lab.

**Comment by ktreese** [comment]
@garethr I upgraded the `gettext-setup` gem to resolve the issue as the version I had installed did not contain the `translation_repositories` feature implemented in [version 0.17](https://github.com/puppetlabs/gettext-setup-gem/commit/0fcb0971faf094b0689bf302b04327a09de41c0e) of that gem.

What I had:
```
➜  tasks gem list | grep gettext-seutp                                                                                                                   
gettext-setup (0.10)
```
What I updated to:
```
➜  tasks sudo gem update gettext-setup                                                                                                                    
Updating installed gems
Updating gettext-setup
Fetching: gettext-setup-0.28.gem (100%)
Successfully installed gettext-setup-0.28
Parsing documentation for gettext-setup-0.28
Installing ri documentation for gettext-setup-0.28
Installing darkfish documentation for gettext-setup-0.28
Done installing documentation for gettext-setup after 0 seconds
Gems updated: gettext-setup
```

Once upgraded, I was able to use the `puppetlabs/package` task module.  With that, I think this problem is actually more specific to use of this particular task module than an issue with bolt as I was able to run, unless this issue will surface for any task written in ruby?

**Comment by ktreese** [comment]
I'm closing this as I agree it probably doesn't belong here.  My goal was more to provide a solution for those who will come across this problem when running through use of the package task.
